packages/notebook/src/default-toolbar.tsx

```diff
@@ -191,6 +216,10 @@ export namespace ToolbarItems {
           sessionDialogs
         )
       },
+      {
+        name: 'restart-and-run',
```

Comment:
Can we be a bit more descriptive?
```suggestion
        name: 'restart-and-run-all',
```